http/request/ContactUsRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+ 
+namespace  App\Http\Request;
```

Comment:
Folder name should be called `Request` and not `request` it matters.

Reply:
I changed request into Request

Reply:
changed folder name in project, but I can't change it in git. /home/miron/bootcamp-project/app/Http/Request/ContactUsRequest.php

Reply:
if you use linux than you should not have this problem. In other case, I recommend to move file remove folder and create it again copying this file.